install-panel.sh

```diff
@@ -136,35 +136,16 @@ required_input() {
 }
 
 password_input() {
-  local  __resultvar=$1
-  local  result=''
-
-  while [ -z "$result" ]; do
-    echo -n "* ${2}"
-
-    # modified from https://stackoverflow.com/a/22940001
-    while IFS= read -r -s -n1 char; do
-      [[ -z $char ]] && { printf '\n'; break; } # ENTER pressed; output \n and break.
-      if [[ $char == $'\x7f' ]]; then # backspace was pressed
-          # Only if variable is not empty
-          if [ -n "$result" ]; then
-            # Remove last char from output variable.
-            [[ -n $result ]] && result=${result%?}
-            # Erase '*' to the left.
-            printf '\b \b' 
-          fi
-      else
-        # Add typed char to output variable.
-        result+=$char
-        # Print '*' in its stead.
-        printf '*'
-      fi
+  local result=$1
+  echo -e "Please enter the password that is used throughout the install. (Use something strong)"
+  while [[ -z "$passOriginal" || ! "$passOriginal" == "$passConfirm" ]]
+    do
+      read -rsp "Password: " passOriginal && echo
+      read -rsp "Confirm Password: " passConfirm && echo
```

Comment:
Why do you use `&& echo` at the end? Also, what's the thing with the `-p` prompt? From `man read` I can see that `-p` is described as.

```
Display prompt on standard error, without a trailing newline, before attempting to read any input.  The prompt is displayed only if input is coming from a terminal.
```

Reply:
I think the -p is doesn't display anything as text but it allows text to be entered, Not sure tho
the echo is just adding a new line once you enter an input